packages/polymer-legacy-adapter/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@vaadin/polymer-legacy-adapter",
+  "version": "22.0.0-alpha6",
+  "description": "polymer-legacy-adapter",
+  "main": "index.js",
+  "module": "index.js",
+  "repository": "vaadin/web-components",
+  "keywords": [
+    "Vaadin",
+    "web-components",
+    "web-component",
+    "polymer"
+  ],
+  "author": "Vaadin Ltd",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/vaadin/web-components/issues"
+  },
+  "homepage": "https://vaadin.com/elements",
+  "files": [
+    "*.d.ts",
+    "vaadin-*.js"
```

Comment:
Please change this to use `*.js` file here to make sure `index.js` and `style-modules.js` are included.

Reply:
Good catch. Done.